pkg/datapath/loader/loader.go

```diff
@@ -205,6 +205,10 @@ func netdevRewrites(ep datapath.EndpointConfiguration, lnc *datapath.LocalNodeCo
 		}
 	}
 
+	if option.Config.EnableUnknownProtocols {
```

Comment:
Could this be put into `LocalNodeConfiguration` instead?

Also, here and below:

```suggestion
	cfg.EnableUnknownProtocols = option.Config.EnableUnknownProtocols
```